_posts/2020-06-07-higher-order-hydras.markdown

```diff
@@ -0,0 +1,491 @@
+
+I have had to make one major revision to the rules so far because the rules as written didn't do what I thought they were doing.
+Do let me know if it's doing something weird, because that's probably not intentional and it was a legitimate mistake when translating the idea to hard rules.
+
+</div>
+
+# Ordinal notation
+{: #amiko-hydras-ordinals}
+
+{% include collapser.markdown %}
+
+<div>
+
+Hydras naturally yield ordinal notations, so long as you abide by the normal form rules.
+In fact, if you really wanted to, you could use hydras directly to denote ordinals.
+A hydra \\\(A\\\) is less than a hydra \\\(B\\\) if they are not identical and \\\(A\\\) can be reached from \\\(B\\\) by some series of steps.
+This ordering is a well-order with order type whatever the hydra system's ordinal is.
+Amiko hydras follow this too, and they do correspond to some large ordinals.
+
+However, even if Amiko hydras themselves are already an ordinal notation, this is not quite complete.
+If we look at Buchholz hydras, the hydras can be converted into expressions using Buchholz's \\\(\psi\\\), which is an ordinal collapsing function defined just using ordinals.
+I have not yet found or created such a system for Amiko hydras, an ordinal notation defined purely using ordinals which is able to exactly mirror the structure of Amiko hydras.
+It seems challenging, and perhaps that is an item best left for future research.
+
+</div>
+
+# Flattening of Amiko hydras
+{: #amiko-hydras-flattening}
+
+{% include collapser.markdown %}
+
+<div>
+
+For some purposes, it may be useful to flatten the hydra.
+If hydra labels are "internal" and children are "external", in the process of deriving Amiko hydras from the nested hydra system, we externalized that nesting already.
+We can also externalize the hydra labels in Amiko hydras by moving the hydra from the label into a fake first child, except for the root (and sub-roots) which will have no label.
+Rules would need to be slightly reworded so rules about labels instead refer to the first child and rules about children ignore the first child, but the hydras would be functionally the same, just now in tree form without further nesting.
+If you wanted to flatten it further, you could use any of a number of methods which represents a tree uniquely using a list.
+
+</div>
+
+# Fast-growing function and reference number
+{: #amiko-hydras-fast-growing}
+
+{% include collapser.markdown %}
+
+<div>
+
+## Summary of the fast-growing function and reference number
+{: #amiko-hydras-fast-growing}
+
+<div>
+
+For completeness, and for fun, I should define a fast-growing function based on Amiko hydras.
```

Comment:
I think you should change "I should define" to "I am defining".

Reply:
I can still say "I should define" even if I've already defined it. It also works I think because the reader hasn't been introduced to AH yet and I wanted to walk them through it.

Reply:
Fair enough, this was just a suggest for style 👍